### DISCUSSION TITLE
Are there any plans of making web app of this project which we can run with docker and make things publicly accessible too

### DISCUSSION THREAD

**Discussion by Greatz08** [chidiwilliams/buzz#714] [General]
As you know there are alot of awesome selfhosted solutions which we can run and operate over web and at same time share with others too easily so was thinking if there are any current or future plans for it. The Reason for me wanting web app was that i was thinking to run this on my server rather than feeding my own valuable resources to whisper running locally so just wanted to make user of my server resources for this as it would make much easier to access as max resides over web/browser and to open one tab and just push file and get transcript becomes easier plus if more things can be added to that web app like history of generation of text plus additon of llm api which can be used to automatically summarize the transcript makes things more useful.
Btw thanks for this great solution :-))

**Comment by raivisdejus** [comment]
Buzz uses different Whisper backends to transcribe. Added value of Buzz is that it is a nice UI to interact with whisper. You can run each of the whisper backends on server or with docker, check for projects that provide docket containers for those whispers.

This is simple API for whisper https://github.com/morioka/tiny-openai-whisper-api
For CPU server google "whisper cpp docker" or for GPU/any server google more generally "whisper docker" there are several options available
